### ISSUE TITLE
SIGSEGV: segmentation violation

### ISSUE THREAD

**Issue by Fank** [moby/moby#32780]
**Description**

`segmentation violation` killed whole docker infrastructure.
EDIT: After some research: after the 4. so the 5. retry (top 4 lines in syslog) the docker died.

**Steps to reproduce the issue:**
1. Sorry but i don't know, it just died
2.
3.

Some background info:
I added the "infinit" volume driver, after some time i removed all volumes from the services (maybe forgot some) and removed infinit. Later docker crashed.

**Describe the results you received:**
Syslog:
```
Apr 22 22:10:06 Ubuntu-1604-xenial-64-minimal dockerd[9118]: time="2017-04-22T22:10:06.145737527+02:00" level=warning msg="Unable to connect to plugin: /run/docker/plugins/infinit.sock/VolumeDriver.Get: Post http://%2Frun%2Fdocker%2Fplugins%2Finfinit.sock/VolumeDriver.Get: dial unix /run/docker/plugins/infinit.sock: connect: no such file or directory, retrying in 1s"
Apr 22 22:10:07 Ubuntu-1604-xenial-64-minimal dockerd[9118]: time="2017-04-22T22:10:07.146019074+02:00" level=warning msg="Unable to connect to plugin: /run/docker/plugins/infinit.sock/VolumeDriver.Get: Post http://%2Frun%2Fdocker%2Fplugins%2Finfinit.sock/VolumeDriver.Get: dial unix /run/docker/plugins/infinit.sock: connect: no such file or directory, retrying in 2s"
Apr 22 22:10:09 Ubuntu-1604-xenial-64-minimal dockerd[9118]: time="2017-04-22T22:10:09.146372356+02:00" level=warning msg="Unable to connect to plugin: /run/docker/plugins/infinit.sock/VolumeDriver.Get: Post http://%2Frun%2Fdocker%2Fplugins%2Finfinit.sock/VolumeDriver.Get: dial unix /run/docker/plugins/infinit.sock: connect: no such file or directory, retrying in 4s"
Apr 22 22:10:13 Ubuntu-1604-xenial-64-minimal dockerd[9118]: time="2017-04-22T22:10:13.146697228+02:00" level=warning msg="Unable to connect to plugin: /run/docker/plugins/infinit.sock/VolumeDriver.Get: Post http://%2Frun%2Fdocker%2Fplugins%2Finfinit.sock/VolumeDriver.Get: dial unix /run/docker/plugins/infinit.sock: connect: no such file or directory, retrying in 8s"
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: panic: runtime error: invalid memory address or nil pointer dereference
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x9613db]
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: goroutine 96595 [running]:
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: panic(0x166ecc0, 0xc420010070)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/local/go/src/runtime/panic.go:500 +0x1a1
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/volume/store.lookupVolume(0xc42161fc38, 0x7, 0xc420a97ae0, 0x18, 0x7, 0x0, 0x2423700, 0xc421bcaae0)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/volume/store/store.go:545 +0x2cb
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/volume/store.volumeExists(0x24236a0, 0xc420e540f0, 0xc42161fc38, 0x7, 0x1)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/volume/store/store.go:357 +0x89
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/volume/store.(*VolumeStore).checkConflict(0xc42018d270, 0xc42264c260, 0x18, 0xc4221015a3, 0x7, 0x0, 0x0, 0xc423ee3750, 0x4173f4)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/volume/store/store.go:331 +0xc7
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/volume/store.(*VolumeStore).create(0xc42018d270, 0xc42264c260, 0x18, 0xc4221015a3, 0x7, 0xc42268b650, 0xc42268b620, 0x0, 0x0, 0x5000104, ...)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/volume/store/store.go:380 +0x77
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/volume/store.(*VolumeStore).CreateWithRef(0xc42018d270, 0xc42264c260, 0x18, 0xc4221015a3, 0x7, 0x0, 0x0, 0xc42268b650, 0xc42268b620, 0x0, ...)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/volume/store/store.go:276 +0x136
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/volume/store.(*VolumeStore).Create(0xc42018d270, 0xc42264c260, 0x18, 0xc4221015a3, 0x7, 0xc42268b650, 0xc42268b620, 0x1657820, 0xc42268b140, 0xc423ee3968, ...)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/volume/store/store.go:291 +0x8f
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/daemon.(*Daemon).VolumeCreate(0xc420406200, 0xc42264c260, 0x18, 0xc4221015a3, 0x7, 0xc42268b650, 0xc42268b620, 0xc421bebee0, 0x41392a, 0xc42278563c)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/daemon/create.go:233 +0xae
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/daemon/cluster/executor/container.(*containerAdapter).createVolumes(0xc42268bad0, 0x7f52495c6968, 0xc4229bbd00, 0x0, 0x0)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/daemon/cluster/executor/container/adapter.go:379 +0x24b
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/daemon/cluster/executor/container.(*controller).Prepare(0xc420dcacd0, 0x7f52495c6968, 0xc4229bbd00, 0xc4224bc738, 0x24253e0)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/daemon/cluster/executor/container/controller.go:110 +0xd2
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/vendor/github.com/docker/swarmkit/agent/exec.Do(0x7f52495c6968, 0xc4229bbd00, 0xc42242e000, 0x24253e0, 0xc420dcacd0, 0x0, 0x0, 0x0)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/docker/swarmkit/agent/exec/controller.go:308 +0x995
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/vendor/github.com/docker/swarmkit/agent.(*taskManager).run.func2(0x7f5249615290, 0xc421f1f980, 0x0, 0x0)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/docker/swarmkit/agent/task.go:134 +0x142
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: github.com/docker/docker/vendor/github.com/docker/swarmkit/agent.runctx(0x7f5249615290, 0xc421f1f980, 0xc422785740, 0xc4220938c0, 0xc4229bbfc0)
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/docker/swarmkit/agent/helpers.go:9 +0x55
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: created by github.com/docker/docker/vendor/github.com/docker/swarmkit/agent.(*taskManager).run
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[9118]: #011/usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/docker/swarmkit/agent/task.go:150 +0x5e4
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Unit entered failed state.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Failed with result 'exit-code'.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Service hold-off time over, scheduling restart.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopped Docker Application Container Engine.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: Closed Docker Socket for the API.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopping Docker Socket for the API.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: Starting Docker Socket for the API.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: Listening on Docker Socket for the API.
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal systemd[1]: Starting Docker Application Container Engine...
Apr 22 22:10:21 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:21.396819908+02:00" level=info msg="libcontainerd: new containerd process, pid: 20282"
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.421315656+02:00" level=info msg="Graph migration to content-addressability took 0.00 seconds"
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.421623864+02:00" level=warning msg="Your kernel does not support swap memory limit"
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.421671955+02:00" level=warning msg="Your kernel does not support cgroup rt period"
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.421686071+02:00" level=warning msg="Your kernel does not support cgroup rt runtime"
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.422227596+02:00" level=info msg="Loading containers: start."
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.434705635+02:00" level=info msg="Firewalld running: false"
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal kernel: [83420.369871] IPVS: Creating netns size=2176 id=119
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.652143793+02:00" level=info msg="Removing stale sandbox 6739c3208e09c53c1e635b4cf132b822066698d23924097afb3cf45de90c432c (338b192e951e666f1f8064016128fe227572b3f1b9a97c4a2f0b834caf9f9224)"
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal kernel: [83420.645516] IPVS: Creating netns size=2176 id=120
Apr 22 22:10:22 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:22.939547624+02:00" level=info msg="Removing stale sandbox 762f3eba1fe9d5e7da982287e0b97ec739d6ed356c4835c724abd3b9eb0b9b7c (03552a21be3fb86ff9e1cd083147d375fc95a8208b63017e2f726893d1f58e04)"
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal kernel: [83420.857785] IPVS: Creating netns size=2176 id=121
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:23.149481878+02:00" level=info msg="Removing stale sandbox 7e4da5b2f0f2be0ec0af1de2c164a3016f14e889b6dd97a3c0b1f01ab146a3dd (33d9568985b08f2e69ae4df7afdfa47cbf8011513bbfdc099529cf4e946a304b)"
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal kernel: [83421.110391] IPVS: Creating netns size=2176 id=122
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:23.406769911+02:00" level=info msg="Removing stale sandbox 96533e3ef65f5fe2aa6b28615de78703aaf2c94aeac4ddc52f4e24e5c559605d (ba3c72d64044002a7dba67d768f8075f4ac3e11772fb1619bbef2dba864d3a12)"
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal kernel: [83421.312731] IPVS: Creating netns size=2176 id=123
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:23.599000307+02:00" level=info msg="Removing stale sandbox d86a1d9b96cca191a9458623ec5e28265cd0faa5b7dc988db580b41b584652a9 (ece42b1bc75af3011e0635a56fc4a9081831001390126a4ae828ae87e3d5d70d)"
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal kernel: [83421.515480] IPVS: Creating netns size=2176 id=124
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:23.798754129+02:00" level=info msg="Removing stale sandbox e2297cbdab4152ab2912a4a113edea5b71422a53d2ce1558e6fbf423efdf7ec5 (fe14c8020ad6320f9429cb3a52e83014acd72eb52f1bf51d6a51b2a3ef048611)"
Apr 22 22:10:23 Ubuntu-1604-xenial-64-minimal kernel: [83421.719968] IPVS: Creating netns size=2176 id=125
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:24.001840336+02:00" level=info msg="Removing stale sandbox ingress_sbox (ingress-sbox)"
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal kernel: [83421.790858] docker_gwbridge: port 1(veth2c06a3b) entered disabled state
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal kernel: [83421.793713] br0: port 2(veth2) entered disabled state
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal kernel: [83421.796750] device veth2c06a3b left promiscuous mode
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal kernel: [83421.796752] docker_gwbridge: port 1(veth2c06a3b) entered disabled state
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal kernel: [83421.829443] device veth2 left promiscuous mode
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal kernel: [83421.829446] br0: port 2(veth2) entered disabled state
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal dockerd[20270]: time="2017-04-22T22:10:24.130607471+02:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal dockerd[20270]: Error starting daemon: Error initializing network controller: Error creating default "bridge" network: could not find an available, non-overlapping IPv6 address pool among the defaults to assign to the network
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Main process exited, code=exited, status=1/FAILURE
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: Failed to start Docker Application Container Engine.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Unit entered failed state.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Failed with result 'exit-code'.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Service hold-off time over, scheduling restart.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopped Docker Application Container Engine.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: Closed Docker Socket for the API.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopping Docker Socket for the API.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: Starting Docker Socket for the API.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: Listening on Docker Socket for the API.
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal systemd[1]: Starting Docker Application Container Engine...
Apr 22 22:10:24 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:24.876707985+02:00" level=info msg="libcontainerd: new containerd process, pid: 20669"
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:25.886720686+02:00" level=info msg="Graph migration to content-addressability took 0.00 seconds"
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:25.886908744+02:00" level=warning msg="Your kernel does not support swap memory limit"
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:25.886943519+02:00" level=warning msg="Your kernel does not support cgroup rt period"
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:25.886952924+02:00" level=warning msg="Your kernel does not support cgroup rt runtime"
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:25.887258059+02:00" level=info msg="Loading containers: start."
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:25.893397533+02:00" level=info msg="Firewalld running: false"
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: time="2017-04-22T22:10:25.924146294+02:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal dockerd[20658]: Error starting daemon: Error initializing network controller: Error creating default "bridge" network: could not find an available, non-overlapping IPv6 address pool among the defaults to assign to the network
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Main process exited, code=exited, status=1/FAILURE
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal systemd[1]: Failed to start Docker Application Container Engine.
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Unit entered failed state.
Apr 22 22:10:25 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Failed with result 'exit-code'.
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Service hold-off time over, scheduling restart.
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopped Docker Application Container Engine.
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal systemd[1]: Closed Docker Socket for the API.
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopping Docker Socket for the API.
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal systemd[1]: Starting Docker Socket for the API.
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal systemd[1]: Listening on Docker Socket for the API.
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal systemd[1]: Starting Docker Application Container Engine...
Apr 22 22:10:26 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:26.112347980+02:00" level=info msg="libcontainerd: new containerd process, pid: 20734"
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:27.122110594+02:00" level=info msg="Graph migration to content-addressability took 0.00 seconds"
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:27.122256354+02:00" level=warning msg="Your kernel does not support swap memory limit"
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:27.122277470+02:00" level=warning msg="Your kernel does not support cgroup rt period"
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:27.122283637+02:00" level=warning msg="Your kernel does not support cgroup rt runtime"
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:27.122532522+02:00" level=info msg="Loading containers: start."
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:27.127411036+02:00" level=info msg="Firewalld running: false"
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: time="2017-04-22T22:10:27.159003023+02:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal dockerd[20724]: Error starting daemon: Error initializing network controller: Error creating default "bridge" network: could not find an available, non-overlapping IPv6 address pool among the defaults to assign to the network
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Main process exited, code=exited, status=1/FAILURE
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: Failed to start Docker Application Container Engine.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Unit entered failed state.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Failed with result 'exit-code'.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Service hold-off time over, scheduling restart.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopped Docker Application Container Engine.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: Closed Docker Socket for the API.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: Stopping Docker Socket for the API.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: Starting Docker Socket for the API.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: Listening on Docker Socket for the API.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.service: Start request repeated too quickly.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: Failed to start Docker Application Container Engine.
Apr 22 22:10:27 Ubuntu-1604-xenial-64-minimal systemd[1]: docker.socket: Unit entered failed state.
```

**Describe the results you expected:**
Not crashing

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
Client:
 Version:      17.04.0-ce
 API version:  1.28
 Go version:   go1.7.5
 Git commit:   4845c56
 Built:        Mon Apr  3 18:07:42 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.04.0-ce
 API version:  1.28 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   4845c56
 Built:        Mon Apr  3 18:07:42 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**

```
Containers: 5
 Running: 5
 Paused: 0
 Stopped: 0
Images: 5
Server Version: 17.04.0-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: active
 NodeID: o7q08dab6ekpbnoox4mkjonb0
 Is Manager: true
 ClusterID: pedmkvmgjvqv2px6ncyc0hbq3
 Managers: 1
 Nodes: 1
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
 Node Address: 88.99.195.101
 Manager Addresses:
  88.99.195.101:2377
Runtimes: runc
Default Runtime: runc
Init Binary: 
containerd version: 422e31ce907fd9c3833a38d7b8fdd023e5a76e73
runc version: 9c2d8d184e5da67c95d601382adf14862e4f2228
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.8.0-46-generic
Operating System: Ubuntu 16.04.2 LTS
OSType: linux
Architecture: x86_64
CPUs: 8
Total Memory: 62.72GiB
Name: Ubuntu-1604-xenial-64-minimal
ID: 3UER:4PED:MI3Z:SE5G:3OQX:NYJM:VTIT:ZVYM:DPEO:U2TO:WSNJ:WY4G
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: No swap limit support
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by Fank** [comment]
@cpuguy83 could you please take a look in it?

**Comment by cpuguy83** [comment]
Thanks, this is fixed in 17.05 which should land next week.
RC's are available now.
